internal/operator/cluster/cluster.go

```diff
@@ -85,8 +85,13 @@ func AddClusterBase(clientset kubeapi.Interface, cl *crv1.Pgcluster, namespace s
 	// logic following a restart of the container.
 	// If the configmap already exists, the cluster creation will continue as this is required
 	// for certain pgcluster upgrades.
-	if err = operator.CreatePGHAConfigMap(clientset, cl, namespace); err != nil &&
-		!kerrors.IsAlreadyExists(err) {
+	if err = operator.CreatePGHAConfigMap(clientset, cl,
+		namespace); kerrors.IsAlreadyExists(err) {
+		log.Debugf("found existing pgha ConfigMap for cluster %s, setting init flag to 'true'",
+			cl.GetName())
```

Comment:
I would probably put this one at the `Info` level.

Reply:
Changed to `Info`.